### DISCUSSION TITLE
Sneak peek into the future of GitHub and AI - Powered by GitHub Copilot X

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#50713] [Announcements]
Today, we are announcing our vision for AI, powered by [GitHub Copilot X](https://www.github.com/features/preview/copilot-x). 

A new generation of GitHub Copilot will support pull requests, answer your questions using Copilot for Docs, provide in-depth analysis of your code using Chatbot right into your editor, and much more by creating an AI assistant readily available to you. 

Check out this [blog post](https://github.blog/2023-03-22-github-copilot-x-the-ai-powered-developer-experience/) by GitHub CEO, [Thomas Dohmke](https://github.com/ashtom), where he shares a peek into the future of GitHub and AI.

To ask questions, learn from other users, and share product feedback & bugs - head over to the [Copilot category discussion](https://github.com/orgs/community/discussions/50727) 

![Screenshot 2023-03-22 at 7 42 26 PM](https://user-images.githubusercontent.com/95834130/226931354-341cd9c0-0d9f-4d7d-96c8-4abef1948e83.png)

Generative AI models will drive the next wave of innovation—and we want that innovation to start with developers. 

We started our mission with GitHub Copilot, which has transformed how millions of developers build software. But that’s only one piece of software development, there are manifold opportunities to integrate AI into a developer’s daily experience to optimize productivity, minimize mundane tasks, and ultimately increase happiness. [Get GitHub Copilot](https://github.com/features/copilot) today to be a part of the next generation of AI.

Resources to keep you up to date:

- [GitHub Copilot for Docs](https://www.github.com/features/preview/copilot-x#docs)
- [GitHub Copilot for PR](https://www.github.com/features/preview/copilot-x#pullrequests)(Technical Preview)
- [GitHub Copilot Chat](https://www.github.com/features/preview/copilot-x#chat)
- [GitHub Copilot for CLI](https://www.github.com/features/preview/copilot-x#cli) (Technical Preview)
- [Explore all Previews](https://github.com/features/preview) (Be the first to try out GitHub’s new features)
- [GitHub Next](https://githubnext.com/projects/hey-github/)

Let's build from here 🚀

**Comment by JOAQUINPABLO001** [comment]
en realidad eso ya existe hace tiempo verdad?

**Comment by JOAQUINPABLO001** [comment]
me pueden  pasar una distribucion GNU/ linux que sirva y en español con inteligencia artificial

**Comment by chrisblanch39** [comment]
@JOAQUINPABLO001 #5204 ```coffeescript

```

**Comment by chrisblanch39** [comment]
[zipMoney-contract.pdf](https://github.com/community/community/files/11042750/zipMoney-contract.pdf)
50728 

> [Www](gjub.com)

**Comment by Akash1134** [comment]
To ask questions, learn from other users, and share product feedback & bugs - head over to the [Copilot category discussion](https://github.com/orgs/community/discussions/50727)
